test/__snapshot__/parser/declarations/let/let-union.md

```diff
@@ -0,0 +1,105 @@
+# Kataw parser test case
+
+## Input
+
+`````js
+
+      let x:
+      |
+       /* detached comment */ string;
```

Comment:
@aladdin-add Look at this one, and then look at the printed output. Exactly what I was writing about earlier.
It works 100% too, so I guess you did a great job with this detach comment 👍